### PR TITLE
Finish off cleaning the top level namespace of core

### PR DESCRIPTION
**Pull request by rgbkrk**
This cleans up the top level namespace of core (no more `@nteract/core/actions`)

### CODE REVIEW

File: applications/desktop/src/notebook/index.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ import { initGlobalHandlers } from "./global-events";
 
 import { state } from "@nteract/core";
 
+// $FlowFixMe: this state tree is missing core and modals
```
Comment (theengineear) [inline applications/desktop/src/notebook/index.js]: I think you were going to make core/modals optional, right? Can we get rid of some of these `$FlowFixMe`s?

File: applications/desktop/src/notebook/middlewares.js [hunk 2]
```diff
@@ -11,7 +11,10 @@ type Action = {
   type: string
 };
 
-const middlewares = [createEpicMiddleware(rootEpic), errorMiddleware];
+const middlewares = [
+  createEpicMiddleware(rootEpic),
+  coreMiddlewares.errorMiddleware
```
Comment (theengineear) [inline applications/desktop/src/notebook/middlewares.js:16]: 👌

File: packages/core/src/middlewares.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // NOTE: These are just default middlewares here for now until I figure out how
 // to divide up the desktop app and this core package
 
-import * as selectors from "../selectors";
+import * as selectors from "./selectors";
```
Comment (theengineear) [inline packages/core/src/middlewares.js:5]: 🎉

File: packages/core/src/state/old/index.js [hunk 4]
```diff
@@ -212,5 +212,8 @@ export type AppState = {
   app: RecordOf<AppRecordProps>,
   document: RecordOf<DocumentRecordProps>,
   comms: RecordOf<CommsRecordProps>,
-  config: Map<string, any>
+  config: Map<string, any>,
+  // TODO FIXME FIXME
+  core: Map<any, any>,
+  modals: Map<any, any>
```
Comment (theengineear) [inline packages/core/src/state/old/index.js:218]: 😆

Comment (rgbkrk) [reply packages/core/src/state/old/index.js:218]: Yeahhhhhhh

**Pull request merged**